packages/synapse-react-client/src/lib/containers/oauth/CreateOAuthClient.tsx

```diff
@@ -278,6 +254,7 @@ export const CreateOAuthModal: React.FunctionComponent<
                 </InputAdornment>
               ),
             }}
+            margin={idx === 0 ? 'normal' : 'dense'}
```

Comment:
Reduce margin for subsequent redirection URI fields, so the fields appear closer together